src/puppetlabs/kitchensink/core.clj

```diff
@@ -150,10 +150,9 @@ to be a zipper."
 (defn lines
   "Returns a sequence of lines from the given filename"
   [filename]
-  (-> filename
-    (fs/file)
-    (reader)
-    (line-seq)))
+  (with-open [file-reader (reader (fs/file filename))]
+    ;; line seq is lazy and file-reader gets closed
```

Comment:
This changes the behavior to be non-lazy and will have a different memory footprint than before...  Should this be considered a breaking change to make it more evident to potential adopters?